ui/src/components/filter/components/Settings.vue

```diff
@@ -36,6 +36,7 @@
 <style lang="scss">
 .settings-dropdown {
     width: 200px;
+    background-color: var(--bs-grey-300) !important;
```

Comment:
```suggestion
    background-color: var(--bs-gray-300) !important;
```

Reply:
**Need to check the bg color of switch in off for light theme.**
![image](https://github.com/user-attachments/assets/a45f2d20-d4b6-4500-be8a-135cbb821ae3)

border need to checked with gray-300.
![image](https://github.com/user-attachments/assets/143f27ad-9a48-4c9e-8797-d9b5957a6add)

Maybe arrow should also be in gray-300
![image](https://github.com/user-attachments/assets/99c9e8e8-28f1-4b2d-9bfb-e153d1a7ce13)

Reply:
yes , firstly in the previous conversation , it was decided to first change the color and then we will plan how it is going to be in final .